InnerEye/ML/reports/notebook_report.py

```diff
@@ -56,6 +56,25 @@ def print_header(message: str, level: int = 2) -> None:
     display(Markdown(f"{prefix} {message}"))
 
 
+def print_table(rows: Sequence[Sequence[str]], header: Optional[Sequence[str]] = None) -> None:
```

Comment:
Effectively you are creating a table from a dataframe here. I think there's already a built-in way of doing that with notebooks?

Reply:
yes, good point. looks a bit nicer, too 🙂